libraries/botbuilder-core/src/testAdapter.ts

```diff
@@ -370,34 +478,31 @@ export class TestAdapter extends BotAdapter implements ExtendedUserTokenProvider
      */
     public async getUserToken(context: TurnContext, connectionName: string, magicCode?: string): Promise<TokenResponse> {
         const key: UserToken = new UserToken();
-        key.ChannelId = context.activity.channelId;
-        key.ConnectionName = connectionName;
-        key.UserId = context.activity.from.id;
+        key.channelId = context.activity.channelId;
+        key.connectionName = connectionName;
+        key.userId = context.activity.from.id;
 
         if (magicCode) {
-            var magicCodeRecord = this._magicCodes.filter(x => key.EqualsKey(x.Key));
-            if (magicCodeRecord && magicCodeRecord.length > 0 && magicCodeRecord[0].MagicCode === magicCode) {
+            const magicCodeRecord = this._magicCodes.filter(x => key.equalsKey(x.key));
```

Comment:
It looks like you're filtering and then extracting just the first element. Consider `Array.prototype.find` instead.